subprojects/core/src/main/java/org/gradle/initialization/DefaultSettings.java

```diff
@@ -171,6 +171,7 @@ public void include(String... projectPaths) {
     }
 
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
I'd replace this suppression with `@Deprecated`. This passes the `sanityCheck` as well and I think is clearer.
Also, with the suppression, Idea complains that it is redundant, but just removing it will fail the `sanityCheck`.

Reply:
Makes sense. `@Deprecated` is in place now.